### PR TITLE
Update CLI documentation

### PR DESCRIPTION
**Pull request by nick-drozd**
There are no doubt gaps and irregularities here, but this PR should bring the CLI documentation to a mostly complete state. Reviewers should go through their favorite sections and look for errors and other areas where improvements can be made.

### CODE REVIEW

File: cli/sawtooth_cli/identity.py [hunk 1]
```diff
@@ -58,29 +58,47 @@ def add_identity_parser(subparsers, parent_parser):
...
-                                             dest="subcommand")
     identity_parsers.required = True
 
-    policy_parser = identity_parsers.add_parser('policy')
+    # policy
+    policy_parser = identity_parsers.add_parser(
+        'policy',
+        help='Provides subcommands to display existing policies and create '
+        'new policies',
+        description='This subcommand is used to list the current policy '
```
Comment (agunde406) [inline cli/sawtooth_cli/identity.py]: policy -> policies

File: cli/sawtooth_cli/identity.py [hunk 2]
```diff
@@ -127,16 +147,26 @@ def add_identity_parser(subparsers, parent_parser):
         choices=['default', 'csv', 'json', 'yaml'],
         help='the format of the output')
 
-    role_parser = identity_parsers.add_parser('role')
+    # role
+    role_parser = identity_parsers.add_parser(
+        'role',
+        help='Provides subcommands to display existing policies and create '
+        'new roles',
```
Comment (agunde406) [inline cli/sawtooth_cli/identity.py]: policies -> roles

File: docs/source/cli/poet.rst [hunk 3]
```diff
@@ -19,23 +19,43 @@
...
+Sawtooth with the PoET consensus method. The genesis subcommand
+creates a Batch for the genesis Block. The enclave subcommand
+generates enclave setup information.
+
 poet
 ====
 
+The poet command generates information for configuring a node to use
+Sawtooth with the PoET consensus method. The genesis subcommand
+creates a Batch for the genesis block. The enclave subommand generates
```
Comment (agunde406) [inline docs/source/cli/poet.rst]: subommand -> subcommand

File: docs/source/cli/validator.rst [hunk 4]
```diff
@@ -19,8 +19,50 @@
...
+
+- Use the ``--peering`` option to set the peering type to ``dynamic``
+  or ``static``.
+
+  - If set to ``static``, use the ``--peers`` option to list the URLs
+    of all peers that the validator should connect to, using the
+    format tcp://hostname:port. Specify multiple peer URLs in a
+    comma-separated list.
+
+    - If set to ``dynamic``, any static peers will be processed first,
```
Comment (agunde406) [inline docs/source/cli/validator.rst]: Weird indention

**Comment by chenette** [comment]
@peterschwarz - Yes, the capitalization inconsistencies are a known issue.  I plan to fix them (and other minor problems like punctuation, etc.) in a later edit pass.

File: docs/source/cli/poet.rst [hunk 5]
```diff
@@ -19,23 +19,41 @@
 PoET CLI
 ********
 
+The PoET CLI initializes the proof-of-concept consensus mechanism for
+Sawtooth by generating enclave setup information and creating a Batch
+for the genesis block. For more information, see :doc:/architecture/poet.
```
Comment (chenette) [inline docs/source/cli/poet.rst]: Sorry - I gave you bad RST code in my first review.  It needs backticks:
     :doc:`/architecture/poet`

If this doesn't work, copy the doc rev in app_developers_guide/ubuntu.rst.

File: docs/source/cli/poet.rst [hunk 6]
```diff
@@ -19,23 +19,41 @@
...
+
+The poet command generates information for configuring a node to use
```
Comment (chenette) [inline docs/source/cli/poet.rst]: Delete this paragraph.  Sorry - that was my mistake in the google doc (it belongs only in the next "poet" subsection).

**Pull request merged**